v0/destinations/yahoo_dsp/transform.js

```diff
@@ -0,0 +1,197 @@
+const sha256 = require("sha256");
+const { BASE_ENDPOINT, ENDPOINTS } = require("./config");
+const {
+  defaultRequestConfig,
+  CustomError,
+  isDefinedAndNotNullAndNotEmpty,
+  defaultPutRequestConfig,
+  removeUndefinedAndNullValues,
+  getSuccessRespEvents,
+  getErrorRespEvents
+} = require("../../util");
+
+const {
+  getAccessToken,
+  populateIncludes,
+  populateExcludes
+} = require("./util");
+
+const populateIdentifiers = (attributeArray, { Config }) => {
```

Comment:
Please add js doc, also what are the structure of attribute array

Reply:
Updated it. It's showing up.